src/CSModule.sol

```diff
@@ -150,104 +151,54 @@ contract CSModule is
     }
 
     /// @inheritdoc ICSModule
-    function addNodeOperatorETH(
-        uint256 keysCount,
-        bytes calldata publicKeys,
-        bytes calldata signatures,
+    function createNodeOperator(
+        address from,
         NodeOperatorManagementProperties calldata managementProperties,
-        bytes32[] calldata eaProof,
         address referrer
-    ) external payable whenResumed {
-        (uint256 nodeOperatorId, uint256 curveId) = _createNodeOperator(
-            managementProperties,
-            referrer,
-            eaProof
-        );
-
-        if (
-            msg.value < accounting.getBondAmountByKeysCount(keysCount, curveId)
-        ) {
-            revert InvalidAmount();
-        }
-
-        accounting.depositETH{ value: msg.value }(msg.sender, nodeOperatorId);
+    )
+        external
+        onlyRole(CREATE_NODE_OPERATOR_ROLE)
+        whenResumed
+        returns (uint256 nodeOperatorId)
+    {
+        if (from == address(0)) revert ZeroSenderAddress();
 
-        _addKeysAndUpdateDepositableValidatorsCount(
-            nodeOperatorId,
-            keysCount,
-            publicKeys,
-            signatures
-        );
-    }
+        nodeOperatorId = _nodeOperatorsCount;
+        NodeOperator storage no = _nodeOperators[nodeOperatorId];
 
-    /// @inheritdoc ICSModule
-    function addNodeOperatorStETH(
-        uint256 keysCount,
-        bytes calldata publicKeys,
-        bytes calldata signatures,
-        NodeOperatorManagementProperties calldata managementProperties,
-        ICSAccounting.PermitInput calldata permit,
-        bytes32[] calldata eaProof,
-        address referrer
-    ) external whenResumed {
-        (uint256 nodeOperatorId, uint256 curveId) = _createNodeOperator(
-            managementProperties,
-            referrer,
-            eaProof
-        );
+        no.managerAddress = managementProperties.managerAddress == address(0)
+            ? from
+            : managementProperties.managerAddress;
+        no.rewardAddress = managementProperties.rewardAddress == address(0)
+            ? from
+            : managementProperties.rewardAddress;
+        if (managementProperties.extendedManagerPermissions)
+            no.extendedManagerPermissions = managementProperties
+                .extendedManagerPermissions;
 
-        uint256 amount = accounting.getBondAmountByKeysCount(
-            keysCount,
-            curveId
-        );
-        accounting.depositStETH(msg.sender, nodeOperatorId, amount, permit);
+        unchecked {
+            ++_nodeOperatorsCount;
+        }
 
-        _addKeysAndUpdateDepositableValidatorsCount(
+        emit NodeOperatorAdded(
```

Comment:
Should we add extendedManagerPermissions here?

Reply:
we can, but it adds a complexity for UIs, dashboards, etc
not sure yet we should